Documentation/upgrade.md

```diff
@@ -202,20 +214,45 @@ kubectl -n rook get pod ${API_POD_NAME} -o jsonpath='{.status.phase}{"\n"}{.spec
 Remember to verify the cluster health using the instructions found in the [health verification section](#health-verification).
 
 ### Monitors
-There are multiple monitor pods to upgrade and they are each individually managed by their own replica set.
-**For each** monitor's replica set, you will need to update the pod template spec's image version field to `rook/rook:v0.5.1`.
-For example, we can update the replica set for `mon0` with:
+#### Upgrading from 0.6.x or lower
+If upgrading from a version of Rook of 0.6.x, the upgrade of monitor pods is done differently. In 0.6.x or lower, monitor pods were being managed by a `replicaset`.
+To upgrade the monitor pods to the new version, we rely on the operator's monitor fail over feature.
```

Comment:
nit: failover is commonly one word

Reply:
In visual code, i was getting a green scribble line signifying that the spelling was wrong. But i trust you and your english.

Reply:
maybe it is just my english. :) Two words works too.